frontend/packages/console-dynamic-plugin-sdk/src/extensions/cluster-overview.ts

```diff
@@ -50,6 +50,18 @@ export type ClusterOverviewMultilineUtilizationItem = ExtensionDeclaration<
   }
 >;
 
+/** Adds an item to the Details card of Overview Dashboard */
+export type ClusterOverviewDetailItem = ExtensionDeclaration<
+  'console.dashboards/overview/detail/item',
+  {
+    /** The item title */
+    id?: string;
```

Comment:
why is this needed ?

Reply:
So far just for the react 's `key` prop. I was thinking about passing generic props down to the component, but this can be added later.

Reply:
Not anymore with the `uid`